### ISSUE TITLE
tool issue

### ISSUE THREAD

**Issue by blood607** [projectdiscovery/httpx#1952]
"Use with caution. You are responsible for your actions.
Developers assume no liability and are not responsible for any misuse or damage."
This is showing whenever I launched the tool.

**Comment by dogancanbakir** [comment]
Yes, this is the disclaimer we print with every run. What's the issue you're having? Could you provide a bit more detail? Like, cmd and the full output.
